src/ykfde-enroll

```diff
@@ -146,24 +164,79 @@ if [ "$YKFDE_RESPONSE" ]; then
   fi
 fi
 
-echo "Please provide the old LUKS passphrase for the existing keyslot."
-printf " Enter passphrase: "
-if [ "$DBG" ]; then read -r YKFDE_OLD_PASSPHRASE; else read -r -s YKFDE_OLD_PASSPHRASE; fi
+if [ "$YKFDE_SLOT_KILL" ]; then
+  [ "$DBG" ] && printf '%s\n' " > Killing with 'cryptsetup luksKillSlot $YKFDE_LUKS_DEV $LUKS_KEYSLOT'..." || echo " > Killing slot with 'cryptsetup'..."
+  printf %s "$YKFDE_PASSPHRASE" | cryptsetup luksKillSlot "$YKFDE_LUKS_DEV" "$LUKS_KEYSLOT" 2>&1
+  printf '%s\n' "   LUKS key slot $LUKS_KEYSLOT successfully killed"
+  exit 0
```

Comment:
@Vincent43 Exit code 0 is correct here, because it is successful.

Reply:
You're right. My mistake :smile:.